### ISSUE TITLE
reranker（bge-reranker-large） loss compute problem

### ISSUE THREAD

**Issue by Jeremyywb** [FlagOpen/FlagEmbedding#1112]
作者你好，我研读源码时发现一个比较疑惑的问题，想请教一下。

- 问题所在代码路径是这里：
https://github.com/FlagOpen/FlagEmbedding/blob/master/FlagEmbedding/reranker/modeling.py

- 具体的问题出现在代码中，计算loss部分的：
   代码初始注册了一个target_label，是一个长为batch_size，且全为0，的一维tensor。
   计算loss时候所用代码为
    ```python
    loss = self.cross_entropy(scores, self.target_label)
   ```
  - 其中target_label: 代码初始化时，注册了一个target_label，是一个长为batch_size，且全为0，的一维tensor。
  - scores是一个，size为（batch_size, train_group_size的label）
 - 我问题是：
   target_label不应该是一个scores相同shape，且第一个为第一列为1，其余为0的tensor么？

**Comment by staoxiao** [comment]
@Jeremyywb ，0指的是这样本的位置，训练目标是最大化该位置的概率，可以参考torch官方文档对cross-entropy损失的描述：https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss

**Comment by LawsonAbs** [comment]
所以其实bge-reranker-large 的损失函数就是交叉熵，而不是对比学习的那种形式咯
